src/Components/Calculator.js

```diff
@@ -1,40 +1,44 @@
-/* eslint-disable react/prop-types */
-import React from 'react';
+import React, { useState } from 'react';
+import Button from './Button';// Adjust the path based on your project structure
 import '../style/ButtonStyles.css';
+import calculate from '../logic/calculate'; // Adjust the path to calculate.js
 
 function Calculator() {
+  const [calculatorState, setCalculatorState] = useState({
+    total: null,
+    next: null,
+    operation: null,
+  });
+
+  const handleClick = (buttonName) => {
+    const newState = calculate(calculatorState, buttonName);
+    setCalculatorState(newState);
+  };
+
   return (
     <div className="container">
-      <div className="header">0</div>
-      <Button className="button" label="AC" />
-      <Button className="button" label="+/-" />
-      <Button className="button" label="%" />
-      <Button className="button" label="/" />
-      <Button className="button" label="7" />
-      <Button className="button" label="8" />
-      <Button className="button" label="9" />
-      <Button className="button operator" label="x" />
-      <Button className="button" label="4" />
-      <Button className="button" label="5" />
-      <Button className="button" label="6" />
-      <Button className="button operator" label="-" />
-      <Button className="button" label="1" />
-      <Button className="button" label="2" />
-      <Button className="button" label="3" />
-      <Button className="button operator" label="+" />
-      <Button className="button zero" label="0" />
-      <Button className="button" label="." />
-      <Button className="button operator" label="=" />
+      <div className="header">{calculatorState.next || calculatorState.total || '0'}</div>
```

Comment:
- [ ] [optional] @ZabihullahNooriWardak It would have been nice if you increased the size of the output from the calculator, it's currently too small.
![image](https://github.com/ZabihullahNooriWardak/Math-Magicians/assets/61408860/ebed3f12-ea58-4e7d-95bf-3f6eca0be55a)